Cargo.toml

```diff
@@ -89,8 +96,20 @@ cob_sickle_macros = { path = "crates/sickle_macros", version = "0.6.0" }
 cob_sickle_ui_scaffold = { path = "crates/sickle_ui_scaffold", version = "0.6.0" }
 
 [dev-dependencies]
-bevy = { version = "0.15.0", default-features = true, features = ["serialize"] }
+bevy = { version = "0.15", default-features = false, features = ["serialize"] }
```

Comment:
Don't think this works without any bevy features, have you tested this? `cargo check` should do it.